CRM/MembershipExtras/Upgrader.php

```diff
@@ -9,8 +9,11 @@
  */
 class CRM_MembershipExtras_Upgrader extends CRM_MembershipExtras_Upgrader_Base {
 
-  public function Install() {
+  public function Install()
+  {
```

Comment:
Please open the brackets on the same line as the function's name definition.

eg.
```php
public function Install() {
  ...
}
```